src/drivers/px4io/px4io.cpp

```diff
@@ -1691,14 +1693,23 @@ PX4IO::io_handle_status(uint16_t status)
 	/**
 	 * Get and handle the safety status
 	 */
-	safety_s safety{};
-	safety.timestamp = hrt_absolute_time();
-	safety.safety_switch_available = true;
-	safety.safety_off = (status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF) ? true : false;
-	safety.override_available = _override_available;
-	safety.override_enabled = (status & PX4IO_P_STATUS_FLAGS_OVERRIDE) ? true : false;
-
-	_to_safety.publish(safety);
+	const bool safety_off = (status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF) ? true : false;
```

Comment:
```suggestion
	const bool safety_off = status & PX4IO_P_STATUS_FLAGS_SAFETY_OFF;
```

Reply:
Blind copy pasta. Thanks for actually reviewing.

Reply:
Fixed, squashed, and rebased.